Code Samples/BoxConsoleSample/Objects/box.h

```diff
@@ -0,0 +1,20 @@
+#ifndef BOX_H
+#define BOX_H
+
+/**
+ * Box object
+ * Defines Box and its properties
+ */
+class Box {
```

Comment:
all members are public, so it could be struct

Reply:
changed